src/modules/Dashboard/features/Respondents/Popups/EditRespondentPopup/EditRespondentPopup.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { waitFor, screen, fireEvent } from '@testing-library/react';
+import axios from 'axios';
+
+import { renderWithProviders } from 'shared/utils/renderWithProviders';
+
+import { EditRespondentPopup } from '.';
+
+const onCloseMock = jest.fn();
+const successFakeRequest = () => new Promise((res) => res(null));
+
+const chosenAppletData = {
+  appletId: '12312',
+  respondentSecretId: '12312',
+  respondentId: '12312',
+  respondentNickname: 'respondentNickname',
+  ownerId: '1',
+};
+
+const commonProps = {
+  onClose: onCloseMock,
+  popupVisible: true,
+  chosenAppletData,
+};
+
+describe('EditRespondentPopup component tests', () => {
+  const mockedAxios = axios.create();
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  test('EditRespondentPopup should appear with respondentSecretId and respondentNickname', async () => {
+    renderWithProviders(<EditRespondentPopup {...commonProps} />);
+
+    await waitFor(() => {
+      expect(screen.getByDisplayValue(chosenAppletData.respondentSecretId)).toBeInTheDocument();
+      expect(screen.getByDisplayValue(chosenAppletData.respondentNickname)).toBeInTheDocument();
+    });
+  });
+
+  test('EditRespondentPopup should appear success text', async () => {
+    jest.spyOn(mockedAxios, 'post').mockImplementation(successFakeRequest);
```

Comment:
Please consider, it can be used with mocked services and fixtures for responses:
jest-mock-axios or axios-mock-adapter, more details: https://vhudyma-blog.eu/3-ways-to-mock-axios-in-jest/

or you can implement service with fixtures and use mocked request without to do each time:
https://mswjs.io/